metrics/web/lib/dashboard/presentation/widgets/project_groups_dropdown_body.dart

```diff
@@ -26,33 +25,29 @@ class ProjectGroupsDropdownBody extends StatelessWidget {
     return DropdownBody(
       state: data.menuState,
       builder: (context, animation) {
```

Comment:
```suggestion
      builder: (context, _) {
```
Please consider effective dart style lint for [unused callback parameters](https://dart.dev/guides/language/effective-dart/style#prefer-using-_-__-etc-for-unused-callback-parameters).